### PR TITLE
Simplify installfest

### PR DESCRIPTION
**Pull request by menolan**

### CODE REVIEW

File: week-1/articles/installfest.md [hunk 1]
```diff
@@ -5,25 +5,37 @@ Welcome to your  software program!
...
+
+For the Heroku Command Line Tools to work correctly you will need a Heroku user account.
+
+1. Visit https://signup.heroku.com/
+2. Create a new user account
+3. **Optional**: Read about [Getting started with Node.js on Heroku](https://devcenter.heroku.com/articles/getting-started-with-nodejs)
 
 ## Windows Installation
 
 The instructions below will walk you through installing the following tools:
```
Comment (mrpndev) [inline week-1/articles/installfest.md:30]: I would also mention that these tools are for a Win install only. Mac users are further along the document. Matt told me that this was an issue in the past.

File: week-1/articles/installfest.md [hunk 2]
```diff
@@ -5,25 +5,37 @@ Welcome to your  software program!
...
 1. Node.js JavaScript Engine
 2. Git Version Control System
-3. Visual Studio Code Text Editor
-4. Cmder Terminal Emulator
+3. Git Bash Terminal Emulator
+4. Visual Studio Code Text Editor
 5. MongoDB and Mongo Compass GUI
 6. Heroku Command-line Tools
 
-You will also be show how to sign up for a user account on the following services:
```
Comment (mrpndev) [inline week-1/articles/installfest.md:22]: Typo show/shown

File: week-1/articles/installfest.md [hunk 3]
```diff
@@ -39,12 +51,7 @@ You will also be show how to sign up for a user account on the following service
 2. Click on the link for Windows
 3. Run the installer from your downloads folder
```
Comment (mrpndev) [inline week-1/articles/installfest.md:52]: Advise of the version they need to use? Stable vs LTS causes issues with proxies when starting React and I'm sure some other issues [only one I know of in the curriculum currently].

File: week-1/articles/installfest.md [hunk 4]
```diff
@@ -39,12 +51,7 @@ You will also be show how to sign up for a user account on the following service
...
 4. Select the default options
-
-#### Github User Account
-
-1. Visit the Github Signup page at <https://github.com/join>
-2. Create a user account and select a unique user name.
```
Comment (mrpndev) [inline week-1/articles/installfest.md:46]: Unique but professional. I would definitely advise them that it should not be pokemon_maniac66@aol.com (I've had that issue in the past). It should be something you wouldn't be afraid to show your grandma.

**Review by mrpndev — CHANGES_REQUESTED** [review]
Looks great other than the changes I think would be good.

**Review by menolan — COMMENTED** [review]
Added Loom instructions and Paul's suggestions.

**Review by menolan — COMMENTED** [review]
Removed Credly and Feedback dropbox sections

**Pull request merged**